portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/MyBlogTest.testcase

```diff
@@ -0,0 +1,68 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the Blogs Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "Blogs Page",
+				widgetName = "Blogs");
+
+			Navigator.gotoPage(pageName = "Blogs Page");
+		}
+	}
+
```

Comment:
Where it's the tearDown? Please add it following the comments on https://github.com/lucasperj/liferay-portal/pull/370#discussion_r1322083529